src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --focus_color: #00acdc;
+}
+
+html {
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.main_header {
+  display: flex;
+  width: 100%;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
+  padding-inline-start: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: nowrap;
+  flex-direction: row;
```

Comment:
It's the default value. No need to specify it.

Reply:
here i removed `flex-wrap` and `flex-direction` , but if i removed `display:flex` and `list-style` i will have this case
![image](https://user-images.githubusercontent.com/86828809/232859786-55f385a4-5894-4645-8413-9fe7f538bde6.png)